example/index.html

```diff
@@ -7,71 +7,97 @@
 <meta name="author" content="" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <meta name="robots" content="noindex, nofollow" />
+
 <style>
-	button {
-			width: 100px;
-			height: 44px;
-	}
+    button {
+        width: 120px;
+        height: 44px;
+    }
 </style>
-
 </head>
 <body>
   <h1>Notify.js demo</h1>
   <p>Notify.js is a simple wrapper for the Web Notification API. The GitHub repo is <a href="https://github.com/alexgibson/notify.js">here</a>.</p>
-  <p>Click the button below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
-  <button id="my-button">Notify Me!</button>
+  <p>Click the buttons below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
+  <div id="buttons">
+      <button id="btn-api">Notify Me!</button>
+      <button id="btn-service-worker">Notify Me from Service Worker!</button>
```

Comment:
Well, the demos share a bit of code, so having them in the same page is rather convenient. I do want to separate the javascript code out of the HTML file though, since it has become rather big.